Loop/Managers/StatusExtensionDataManager.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  StatusExtensionDataManager.swift
+//  Loop
+//
+//  Created by Bharat Mediratta on 11/25/16.
+//  Copyright © 2016 Nathan Racklyeft. All rights reserved.
+//
+
+import HealthKit
+import UIKit
+import CarbKit
+import LoopKit
+
+final class StatusExtensionDataManager: NSObject {
+    unowned let dataManager: DeviceDataManager
+    private var lastContext: StatusExtensionContext?
```

Comment:
yeah - vestigial. looking forward I should be keeping the last context for comparison purposes but I'll get rid of it for now.